.github/workflows/deploy-npm.yml

```diff
@@ -24,19 +24,22 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v6
-      - name: Install node ${{ inputs.node-version }}
+      - name: Install node 20
```

Comment:
Do we need to install anymore if we're using bun? Do we need it in any of our workflows?